### ISSUE TITLE
Huge light bulb near code action suggestions

### ISSUE THREAD

**Issue by Flygrounder** [hlissner/doom-emacs#4677]
**What did you expect to happen?**
Small light bulb should appear next to code action

**What actually happened?**
Huge light bulb appears instead

**Additional details:**
- [My config](https://gitlab.com/flygrounder/dotfiles/-/tree/master/dotfiles/.doom.d)
- ![Screenshot](https://user-images.githubusercontent.com/16033061/108625424-e94e4980-745b-11eb-93fa-5ad5de539512.png)
- Problem was also reported [on reddit](https://www.reddit.com/r/DoomEmacs/comments/ll60il/massive_lightbulb_shows_up_next_to_error_in_code )

**Steps to reproduce:**
1. Open any source file
2. Navigate to any place, where there is a code action suggested 

**System information:**
<details><pre>
```
SYSTEM  info       (Pop!_OS 20.10 . Linux 5.8.0-7642-generic x86_64)
        shell      /bin/bash
        path       (/usr/local/sbin /usr/local/bin /usr/sbin /usr/bin /sbin /bin /usr/games /usr/local/games /snap/bin /usr/lib/emacs/26.3/x86_64-linux-gnu/)
EMACS   dir        ~/.emacs.d/
        version    (26.3 мар 26, 2020)
        buildopts  --build x86_64-linux-gnu --prefix=/usr --sharedstatedir=/var/lib --libexecdir=/usr/lib --localstatedir=/var/lib --infodir=/usr/share/info --mandir=/usr/share/man --enable-libsystemd --with-pop=yes --enable-locallisppath=/etc/emacs:/usr/local/share/emacs/26.3/site-lisp:/usr/local/share/emacs/site-lisp:/usr/share/emacs/26.3/site-lisp:/usr/share/emacs/site-lisp --with-sound=alsa --without-gconf --with-mailutils --build x86_64-linux-gnu --prefix=/usr --sharedstatedir=/var/lib --libexecdir=/usr/lib --localstatedir=/var/lib --infodir=/usr/share/info --mandir=/usr/share/man --enable-libsystemd --with-pop=yes --enable-locallisppath=/etc/emacs:/usr/local/share/emacs/26.3/site-lisp:/usr/local/share/emacs/site-lisp:/usr/share/emacs/26.3/site-lisp:/usr/share/emacs/site-lisp --with-sound=alsa --without-gconf --with-mailutils --with-x=yes --with-x-toolkit=gtk3 --with-toolkit-scroll-bars 'CFLAGS=-g -O2 -fdebug-prefix-map=/build/emacs-mEZBk7/emacs-26.3+1=. -fstack-protector-strong -Wformat -Werror=format-security -Wall' 'CPPFLAGS=-Wdate-time -D_FORTIFY_SOURCE=2' 'LDFLAGS=-Wl,-Bsymbolic-functions -Wl,-z,relro'
        features   XPM JPEG TIFF GIF PNG RSVG IMAGEMAGICK SOUND GPM DBUS GSETTINGS GLIB NOTIFY ACL LIBSELINUX GNUTLS LIBXML2 FREETYPE M17N_FLT LIBOTF XFT ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 XDBE XIM THREADS LIBSYSTEMD LCMS2
        traits     (gui server-running envvar-file)
DOOM    dir        ~/.doom.d/
        version    (2.0.9 grafted, HEAD -> develop, origin/develop, origin/HEAD 4792593 2021-02-17 14:20:37 -0500)
        font       #<font-spec nil nil Jetbrains Mono nil nil nil nil nil 14 nil nil nil nil>
        theme      doom-moonlight
        modules    (:completion company ivy :ui doom doom-dashboard doom-quit hl-todo modeline neotree ophints (popup +defaults) tabs vc-gutter vi-tilde-fringe workspaces :editor (evil +everywhere) file-templates fold snippets :emacs dired electric undo vc :checkers syntax :tools direnv docker editorconfig (eval +overlay) lookup lsp magit pdf :lang (cc +lsp) emacs-lisp (go +lsp) (haskell +dante) json (javascript +lsp) latex markdown nix org (python +lsp) qt sh (web +lsp) yaml :config (default +bindings +smartparens))
        packages   ((graphviz-dot-mode) (dap-mode) (fish-mode) (groovy-mode))
        unpin      (n/a)
        elpa       (n/a)
```
</pre></details>

**Comment by denistakeda** [comment]
Same for me here. Very destructed. Any suggestions on how to work around while not fixed?

**Comment by Alexander-Miller** [comment]
The issue is known upstream: https://github.com/emacs-lsp/lsp-ui/issues/573

**Comment by hlissner** [comment]
As mentioned in emacs-lsp/lsp-ui#573, this is an image scaling issue on Emacs 26.x. The quick fix is to upgrade to Emacs 27.1 (Doom will be dropping 26.x support later this year anyway, so it'd be a good idea to do this soon).

That said, I've bumped lsp-ui in e858e25, which disables sideline icons by default, and in a3f242d I've enabled it only for 27+ users. In either case, this issue should be "resolved" as far as Doom's users are concerned. Thanks for bringing it to my attention!
